src/modules/extrakeys/tests_impl.h

```diff
@@ -571,6 +571,43 @@ void test_keypair_add(void) {
     secp256k1_context_destroy(verify);
 }
 
+static void test_hsort_is_sorted(int *ints, size_t n) {
+    size_t i;
+    for (i = 1; i < n; i++) {
+        CHECK(ints[i-1] <= ints[i]);
+    }
+}
+
+static int test_hsort_cmp(const void *i1, const void *i2, void *counter) {
+  *(size_t*)counter += 1;
+  return *(int*)i1 - *(int*)i2;
+}
+
+#define NUM 64
+void test_hsort(void) {
+    int ints[NUM] = { 0 };
```

Comment:
These integer tests never hit the `stride > 64` path in the swaps. It may be a good idea to add a test case for wider types (e.g., scalars because we have a cmp function in the code already). 

edit: ah ok, we're doing this later anyway with the pubkeys... So feel free to ignore.

Reply:
ignored :)